Library/Formula/postgresql.rb

```diff
@@ -77,8 +82,16 @@ def install
   end
 
   def post_install
-    unless File.exist? "#{var}/postgres"
-      system "#{bin}/initdb", "#{var}/postgres"
+    system "pkill postgres || true"
+
+    if (var/"postgres").exist?
+      mv var/"postgres", var/"postgres-#{version_suffix}"
+    elsif (var/"postgres-#{version_suffix}").exist?
+      puts ""
```

Comment:
See the exchange between Tim & I earlier. Basically, need to cover this situation, but wasn't sure how to tell Brew 'everything is good here' in a suitable manner. Initially raised a simple message, then considered 'true' but worried the usage of 'true' would impact the wider formula, so settled on a blank echo, essentially. Open to a better solution entirely.

Sent from my iPhone

> On 6 Jan 2015, at 08:02, Mike McQuaid <notifications@github.com> wrote:
> 
> In Library/Formula/postgresql.rb:
> 
> > @@ -77,8 +82,16 @@ def install
> >    end
> >  
> >    def post_install
> > -    unless File.exist? "#{var}/postgres"
> > -      system "#{bin}/initdb", "#{var}/postgres"
> > +    system "pkill postgres || true"
> > +
> > +    if (var/"postgres").exist?
> > +      mv var/"postgres", var/"postgres-#{version_suffix}"
> > +    elsif (var/"postgres-#{version_suffix}").exist?
> > +      puts ""
> ?
> 
> —
> Reply to this email directly or view it on GitHub.
>